qcodes/instrument_drivers/Keysight/keysight_b220x.py

```diff
@@ -0,0 +1,181 @@
+from qcodes import VisaInstrument
+from qcodes.utils.validators import MultiType, Ints, Enum, Lists
+
```

Comment:
> Caution: If the Free connection rule has been specified, ensure multiple input ports are not connected to the same output port. Such configurations can cause damage

Can we implement a check for this in the driver? And smart users can disable the check, while those who "don't know what they are doing" can have this check enabled which might save the live to their measured samples :)

Reply:
I think I will add a warning that shows when initialising the instrument or when doing a reset.
But I am reluctant to add additional checks for that for the following reasons:
 - *Free* mode is the default behaviour after reset. So it seems to be deemed *safe enough* by Keysight.
 - The only way to add these checks is basically to poll the active connections after each command sent.
   - Users can still use the instrument front panel to override what the active connections and the driver will never know until it polls the connections again.
 - If I add checks for that, users might rely on checks to prevent "stupid mistakes". I'd rather have them pay attention themselves.

Reply:
> I'd rather have them pay attention themselves.
can't agree more with that in this particular case :)

thanks for explanation